src/kvirc/kernel/KviDefaultScript.cpp

```diff
@@ -408,11 +408,11 @@ KviDefaultScriptDialog::KviDefaultScriptDialog()
 	QGridLayout * pLayout = new QGridLayout(this);
 
 	QLabel * pLabel = new QLabel(this);
-	pLabel->setText(__tr2qs("You are about to restore the default script.\nAny script changes you have made will be erased if you\nrestore all stuff or \"Clear personal data\" is selected.\n\nDo you wish to proceed?\n"));
+	pLabel->setText(__tr2qs("You are about to restore the default script.\nAny script changes you have made will be erased if you\nrestore \"All\" or \"Clear custom scripts\" is selected.\n\nDo you wish to proceed?\n"));
 	pLayout->addWidget(pLabel,0,0,1,4);
 
 	// Construct the advanced widget
-	m_pAdvanced = new QGroupBox(__tr2qs("Select default script's elements to restore"),this);
+	m_pAdvanced = new QGroupBox(__tr2qs("Select the Items to Restore"),this);
```

Comment:
IMO thats a group box and also the items in there must be capitalized **Apparently"* According to https://techbase.kde.org/Projects/Usability/HIG/Capitalization which was indicated to me in #1631